spec/ics-027-direct-interchain-exchange/ReadMe.md

```diff
@@ -0,0 +1,74 @@
+# Cosmos Direct Interchain Exchange Protocol (DIEP)
+
+
+#### Core Idea:
+
+Asynchronous liquidity available with IBC that is resistant to theft by misbehaving validators. This eliminates the need to concentrate liquidity (limit orders) in a third party blockchain/venue. Price discovery and execution becomes native between Cosmos zones (e.g. N^2 IBC connections for N zones). There is a cost in terms of higher latency from this architecture.
```

Comment:
What does theft of liquidity mean - theft of funds, or just withholding liquidity from some other chain?

Reply:
I think limit orders are not the only component of "liquidity". Custody, matching, settlement are necessary components to handle liquidity. And I think this document suggests huge roles on CosmosHub to provide such important components. These entire set can be called "Exchange".

So I think this doc is about concentrate important roles on route of "liquidity handling, aka exchange" into CosmosHub.